### ISSUE TITLE
Help button doesn't do anything

### ISSUE THREAD

**Issue by jindrichmynarz** [opendatacz/pc-filing-app#12]
![screen shot 2014-07-17 at 14 17 42](https://cloud.githubusercontent.com/assets/198642/3612508/77f4cd30-0dac-11e4-8be0-33e61ccdb30e.png)

"Help" button in the main menu doesn't do anything. Either make it do something or remove it.

**Comment by jakhrkal** [comment]
Button still not responding at Contracting authority's Dasboard and Bidder's My account, Tenders for withdrawn calls and Completed tenders. Add some basic help text to these.

**Comment by jindrichmynarz** [comment]
Add help messages to i18n files, where `TODO` is marked.

``` bash
cd src/main/resources/cz/opendata/tenderstats/i18n
grep TODO `find *`
```

**Comment by jindrichmynarz** [comment]
Fixed in c456301022419ac459fc635fc040ceb57e2af210.
